coalib/coala_delete_orig.py

```diff
@@ -8,12 +8,25 @@
 from coalib.settings.Section import Section
 
 
-def main(section: Section=None):
+def main(log_printer=None, section: Section=None):
```

Comment:
@Makman2 Okay, So I removed the `log_printers` in `coala-delete-orig` because their were some weird problems with it.

I added it again as suggested here : https://github.com/coala/coala/pull/3740#discussion_r101891219